### DISCUSSION TITLE
在 overleaf 中用 \gdutchapter 的话，在旁边的 File outline 中没有显示论文的结构

### DISCUSSION THREAD

**Discussion by Liuqibaa** [sikouhjw/gdutthesis#42] [General]
如题，就是在 overleaf 中使用了 \gdutsection \gdutsubsection 等前面是 \gdut 前缀的话，就不会显示论文结构，但是用 \chapter \section 等则会显示，而且效果貌似是一样的，请问是这样的吗？

**Comment by sikouhjw** [comment]
> 而且效果貌似是一样的，请问是这样的吗？

`\gdutchapter` 等命令的主要目的是方便输入英文标题，你如果想手写
```tex
\enSectionTitle{英文标题}
\chapter{中文标题}
```
也可以。

如果要为了显示论文结构，就得考虑重定义 `\chapter` 等命令，我当时是担心兼容性问题才没这么做，后面再看看。

**Reply by Liuqibaa** [reply]
好的，因为左边目录的重要性，所以我还是采用了您刚刚提到的方式，即用 \enSectionTitle 将英文标题手动打出来了

**Reply by sikouhjw** [reply]
如果将 `\chapter` 变成输入中英文的，在输出目录、参考文献时，会调用 `\chapter`，导致报错，所以单纯的改是不行的，还挺麻烦的。

**Reply by sikouhjw** [reply]
在 https://github.com/sikouhjw/gdutthesis/commit/476060dd70f7b7189ee4bd967972e7f7d1e7288e 里重定义了 `\chapter` 命令，用法和 `\gdutchapter` 一致，可直接替换 cls。
